net-libs/mbedtls/mbedtls-2.25.0.recipe

```diff
@@ -0,0 +1,143 @@
+SUMMARY="Open source, portable, easy to use, readable and flexible SSL library"
+DESCRIPTION="mbed TLS (formerly known as PolarSSL) makes it trivially easy for \
+developers to include cryptographic and SSL/TLS capabilities in their (embedded) \
+products, facilitating this functionality with a minimal coding footprint."
+HOMEPAGE="https://tls.mbed.org/"
+COPYRIGHT="2006-2018 ARM Limited"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ARMmbed/mbedtls/archive/mbedtls-$portVersion.tar.gz"
```

Comment:
with this URL SOURCE_DIR is necessary. with https://github.com/ARMmbed/mbedtls/archive/v2.25.0.tar.gz SOURCE_DIR ist not.

Reply:
Need to check, but I don't think I changed the SOURCE_URI from previous recipe, but will change, much better :+1:

Reply:
On that note, SOURCE_URI maybe isn't needed, but then we need to use SOURCE_FILENAME :) (but granted, the link on github shows v$portVersion)

Reply:
@korli removed the 2.16 with this PR now, should I change the SOURCE_URI still?

Reply:
Thanks. It's OK as is.